astropy/coordinates/representation/tests/test_representation.py

```diff
@@ -2,28 +2,24 @@
 
```

Comment:
Not sure we should move the tests - we don't for things in `units`.

Reply:
Moving back....

Reply:
✅